kalite/facility/settings.py

```diff
@@ -1,5 +1,5 @@
 try:
-    import local_settings
+    from kalite import local_settings
```

Comment:
Yes absolutely, this way of adding custom settings should be avoided by either: A) Creating one's own project that uses kalite's default settings and then overwrites whichever is custom or B) Having some new magic idiom that replaces this one with a module that's not inside kalite!

#2952 has details about the new project and settings layout